charts/evm-rollup/templates/configmap.yaml

```diff
@@ -72,6 +72,7 @@ data:
   {{- end }}
   {{- if not .Values.global.dev }}
   {{- else }}
+  ASTRIA_COMPOSER_GRPC_ADDR: "0.0.0.0:0"
```

Comment:
Yeah, I guess this is fine albeit odd - we don't have anything talking to this gRPC service in the testnet, so it can just bind whatever it wants.